Gemfile

```diff
@@ -2,6 +2,5 @@ source "https://rubygems.org"
 
 gem "jekyll"
 gem "kramdown-parser-gfm"
-gem 'wax_tasks', '~> 1.1'
-gem 'html-proofer', '~> 3.15'
-
+gem "wax_tasks"
+gem "html-proofer"
```

Comment:
2024-03-25T05:47:14.758Z [WARNING]: /root/.rvm/rubies/ruby-3.2.2/lib/ruby/3.2.0/bundler/resolver.rb:290:in `raise_not_found!': Could not find gem 'wax_tasks (~> 1.1)' in locally installed gems. (Bundler::GemNotFound)

To.solve above issue trying version changes